src/styles/colors.css

```diff
@@ -1,23 +1,78 @@
 :root {
-  --palette-primary: #448fea;
-  --palette-primary-hover: #447bc6;
+  /* Action */
   --palette-action: #33ae10;
+  --palette-action-dark: #257f0c;
+  --palette-action-light: #41dd14;
   --palette-action-hover: #328a15;
-  --palette-highlight: #ff8400;
-  --palette-highlight-hover: #ffa800;
+
+  /* Caution */
   --palette-caution: #e85342;
+  --palette-caution-active: #000;
+  --palette-caution-base: #e85342;
```

Comment:
Interesting. Do we have `-base` for all variations? In diff I don't see one for highlight for example. Not sure if I added these, but I don't see the advantage of not having just one. I don't care between `--name` or `--name-base` whatever is easier to backfill is my sense